app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by(params[:id])
```

Comment:
```suggestion
    @user = User.find_by id: params[:id]
```
check trường hợp k tìm thấy @user